Tensile/BuildCommands/AssemblyCommands.py

```diff
@@ -29,13 +29,12 @@ def _linkIntoCodeObject(
     if os.name == "nt":
         # On Windows, the objectFiles list command line (including spaces)
         # exceeds the limit of 8191 characters, so using response file
-
-        responseFile = os.path.join("/tmp", "clangArgs.txt")
+        responseFile = os.path.join(Path.cwd(), "clang_args.txt")
```

Comment:
The current working directory may not be appropriate for temp files.

Reply:
This was the line of code that originally prompted a number of defect reports for Windows build. `/tmp` won't work either, I haven't found an issue with the CWD. Do you have another recommendation? At least the CWD is guaranteed to exist.

Reply:
Well these are temp files so if you are willing to " import tempfile " then get the temp directory.  Or any other helpers to get a full temp path file name you could use,  doesn't have to be clang_args.txt of course.   Or just say create /tmp in cwd if you know it is in build tree.